src/modules/dashboard/components/ExpenditurePage/ExpenditureSettings/ExpenditureSettings.tsx

```diff
@@ -39,8 +39,23 @@ export const MSG = defineMessages({
     id: 'dashboard.ExpenditurePage.ExpenditureSettings.defaultAdvancedOption',
     defaultMessage: 'Advanced payment',
   },
+  split: {
+    id: 'dashboard.ExpenditurePage.ExpenditureSettings.split',
+    defaultMessage: 'Split',
+  },
 });
 
+const expeditureTypes = [
+  {
+    label: MSG.optionAdvanced,
+    value: 'advanced',
+  },
+  {
+    label: MSG.split,
+    value: 'split',
```

Comment:
I would add an enum `ExpenditureTypes` with `advanced` & `split`  and then use them for values here and then as comparison in `ExpenditureForm`